impl/spanner/directory/directory_test.go

```diff
@@ -0,0 +1,240 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package directory
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/keytransparency/core/directory"
+	"github.com/google/keytransparency/impl/spanner/testutil"
+	"github.com/google/trillian/crypto/keyspb"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	ktspanner "github.com/google/keytransparency/impl/spanner"
+	tpb "github.com/google/trillian"
+)
+
+// Verify that Table implements the directory.Storage interface.
+var _ directory.Storage = &Table{}
+
+func NewForTest(ctx context.Context, t *testing.T) (directory.Storage, func(context.Context)) {
+	t.Helper()
+	ddl, err := ktspanner.ReadDDL()
+	if err != nil {
+		t.Fatal(err)
+	}
+	client, cleanup := testutil.CreateDatabase(ctx, t, ddl)
+	s := New(ktspanner.NewForTest(client))
+	done := func(_ context.Context) { cleanup() }
+	return s, done
```

Comment:
You could return `cleanup` function straight away, given that `Context` is not used by any of the tests below.